src/route.shared.ts

```diff
@@ -235,39 +145,7 @@ function resolveRouteFunction(
   }
 }
 
-type PageRoute = {
-  pageRouteFile: string
-  pageRoute: string | Function
-}
-async function loadPageRoutes(): Promise<Record<PageId, PageRoute>> {
```

Comment:
Did something change here or did you just move the code? That's why such code move should live in a separate PR. Not only is it time consuming to compare each line but, more importantly, it's error prone as I may miss some subtle change.

Reply:
Ok I understand punting the change for the sake of readability, so I'll go ahead and do that, but these refactors will need to occur for performance reasons. All client dependencies need to be kept lean. You shouldn't be shoving all functions into one file.